mjpc/tasks/allegro_cube/right_hand.xml

```diff
@@ -7,7 +7,7 @@
   <default>
     <default class="allegro_right">
       <joint axis="0 1 0" damping=".1"/>
-      <position kp="0.25"/>
+      <position kp="0.5"/>
```

Comment:
Do we want this?

Reply:
unclear on what good values of this are. I think there isn't an immediately sensible value outside the context of sim2real performance.